build.sbt

```diff
@@ -17,6 +18,7 @@ lazy val intellijHaskell = (project in file(".")).
     libraryDependencies += guava,
     libraryDependencies += scalaTest,
     libraryDependencies += sprayJson,
+    libraryDependencies += commonsIO,
```

Comment:
@rikvdkleij BTW, where do you find those functions which JetBrains support internally, e.g `copyStreamToFile` and `openStream`, I think there is no Jetbrains official API documentation there 😅